### PR TITLE
A couple living entity mappings

### PR DESCRIPTION
**Pull request by Pyrofab**

### CODE REVIEW

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 1]
```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
...
+		COMMENT <p> The default behaviour spawns a wither rose if {@code adversary} is a {@code WitherEntity}.
+		COMMENT 
+		COMMENT @param adversary the main adversary responsible for this entity's death
+		ARG 1 adversary
+	METHOD method_23329 computeFallDamage (FF)I
+		ARG 1 fallDistance
+		ARG 2 damageMultiplier
+	METHOD method_23328 playBlockFallSound ()V
+	METHOD method_21754 isHoldingLadder ()Z
+		COMMENT @returns {@code true} if this entity should not lose height while in a climbing state
```
Comment (natanfudge) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: I believe it’s @return

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 2]
```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6132 applyArmorToDamage (Lnet/minecraft/class_1282;F)F
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_23883 dropXp ()V
+	METHOD method_23733 onKilledBy (Lnet/minecraft/class_1309;)V
+		COMMENT Performs secondary effects after this mob has been killed.
+		COMMENT 
+		COMMENT <p> The default behaviour spawns a wither rose if {@code adversary} is a {@code WitherEntity}.
```
Comment (natanfudge) [inline mappings/net/minecraft/entity/LivingEntity.mapping:372]: Missing closing tag?

Comment (Pyrofab) [reply mappings/net/minecraft/entity/LivingEntity.mapping:372]: Javadoc never closes `<p>` tags for some reason, check official docs

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 3]
```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
...
+		COMMENT 
+		COMMENT @param adversary the main adversary responsible for this entity's death
```
Comment (liach) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: Cloak bug? Should be comment on arg 1

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 4]
```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
...
+		ARG 1 adversary
+	METHOD method_23329 computeFallDamage (FF)I
+		ARG 1 fallDistance
+		ARG 2 damageMultiplier
+	METHOD method_23328 playBlockFallSound ()V
+	METHOD method_21754 isHoldingLadder ()Z
```
Comment (liach) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: This name sounds to me as if the entity is holding a ladder item. I'd say `isHeldInLadder` is more clear (you can come up with other better names either)

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 5]
```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
...
+		COMMENT @return {@code true} if this entity should not lose height while in a climbing state
+		COMMENT @see net.minecraft.entity.LivingEntity#applyClimbingSpeed(net.minecraft.util.math.Vec3d) 
+	METHOD method_22382 shouldSpawnConsumptionEffects ()Z
```
Comment (liach) [inline mappings/net/minecraft/entity/LivingEntity.mapping:383]: Effects? Status effects or particles?

**Review by liach — CHANGES_REQUESTED** [review]
Good for the most part.

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping:372]: Yeah, html auto closes `<p>` tags when it meets another `<p>` tag

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 5 above)
Comment (Pyrofab) [reply mappings/net/minecraft/entity/LivingEntity.mapping:383]: Particles, it's called right before spawnConsumptionEffects

Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping:383]: eh, confusing name but sure

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 3 above)
Comment (modmuss50) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Yes, you dont need the `@param adversary` bit either

Comment (Pyrofab) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Sadly Enigma breaks on LivingEntity, so I can't use it to reformat the file.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 4 above)
Comment (Pyrofab) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: `isHoldingOntoLadder` ?

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 5 above)
Comment (Pyrofab) [reply mappings/net/minecraft/entity/LivingEntity.mapping:383]: Not like you can "spawn" status effects anyway

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**